d2l-navigation-dropdown-button-custom.js

```diff
@@ -31,6 +27,9 @@ class NavigationDropdownButtonCustom extends DropdownOpenerMixin(LitElement) {
 		`;
 	}
 
+	getOpenerElement() {
+		return this.shadowRoot && this.shadowRoot.querySelector('button');
```

Comment:
Nit for all of these now that optional chaining is allowed:
```suggestion
		return this.shadowRoot?.querySelector('button');
```